flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -1064,3 +1064,70 @@ def test_numerical_errors(app, setup_planning_test_data, solver):
     assert device_constraints[0]["equals"].max() > device_constraints[0]["max"].max()
     assert device_constraints[0]["equals"].min() < device_constraints[0]["min"].min()
     assert results.solver.status == "ok"
+
+
+@pytest.mark.parametrize(
+    "capacity,site_capacity", [("100kW", "300kW"), ("0.1MW", "0.3MW"), (None, None)]
```

Comment:
I'm just wondering whether a space between the number and the unit is also allowed. If so, let's amend one of the case here to cover that explicitly. If not, please just tell me (in a reply here).

Reply:
Good idea. I tested it and works well with the space just fine. I've added the test case to remind ourselves.